packages/rps/src/utils/channelClient.ts

```diff
@@ -275,6 +275,10 @@ export class ChannelClient {
     );
   }
 
+  unSubscribe(notificationName: NotificationName): void {
+    this.events.removeAllListeners('message');
+  }
+

```

Comment:
Note: This isn't currently narrowed to messages with a given `notificationName`.